Source/Utils/WarpXTagging.cpp

```diff
@@ -22,9 +22,9 @@ WarpX::ErrorEst (int lev, TagBoxArray& tags, Real time, int /*ngrow*/)
         for (BoxIterator bi(bx); bi.ok(); ++bi)
         {
             const IntVect& cell = bi();
-            RealVect pos {AMREX_D_DECL((cell[0]+0.5)*dx[0]+problo[0],
-                                       (cell[1]+0.5)*dx[1]+problo[1],
-                                       (cell[2]+0.5)*dx[2]+problo[2])};
+            RealVect pos {AMREX_D_DECL((cell[0]+0.5_rt)*dx[0]+problo[0],
```

Comment:
to me more precise, this is mixing int and real operations and should explicitly cast to:
```C++
(Real(cell[0])+0.5_rt)*dx[0]+problo[0]
```